content/src/content/jcr_root/apps/core/wcm/components/container/v1/container/.content.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jcr:root xmlns:cq="http://www.day.com/jcr/cq/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0"
+    cq:icon="form"
+    cq:isContainer="{Boolean}true"
+    jcr:description="Components Container"
+    jcr:primaryType="cq:Component"
+    jcr:title="Container (v1)"

```

Comment:
To work with correctly with the resizer it will need to have a `sling:resourceSuperType="wcm/foundation/components/responsivegrid"` property too